SPECS/blobfuse2/blobfuse2.spec

```diff
@@ -0,0 +1,86 @@
+%define preview_suffix preview.3	
+%define blobfuse2_version 2.0.0	
+%define blobfuse2_health_monitor bfusemon
+
+Summary:        FUSE adapter - Azure Storage
+Name:           blobfuse2
+Version:        %{blobfuse2_version}.%{preview_suffix}
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/Tools
+URL:            https://github.com/Azure/azure-storage-fuse/
+# Below is the Github URL where blobfuse2 is accessible. This needs to be defined until blobfuse2 GAs since the version
+# string in spec files does not allow - 
+Source0:        https://github.com/Azure/azure-storage-fuse/archive/%{name}-%{blobfuse2_version}-%{preview_suffix}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/Azure/azure-storage-fuse/archive/%{name}-%{version}.tar.gz -O %{name}-%{version}.tar.gz
+#   2. tar -xf %{name}-%{version}.tar.gz
+#   3. cd azure-storage-fuse-%{name}-%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2021-04-26 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %{name}-%{version}-vendor.tar.gz vendor
+#
+#   NOTES:
+#       - You require GNU tar version 1.28+.
+#       - The additional options enable generation of a tarball with the same hash every time regardless of the environment.
+#         See: https://reproducible-builds.org/docs/archives/
+#       - For the value of "--mtime" use the date "2021-04-26 00:00Z" to simplify future updates.
+Source1:        %{name}-%{version}-vendor.tar.gz
+%global debug_package %{nil}
+%define our_gopath %{_topdir}/.gopath
```

Comment:
Linter can be a bit annoying here, but we're trying to keep the macro definitions at the top of the file.

```suggestion
%global debug_package %{nil}

%define our_gopath %{_topdir}/.gopath
%define preview_suffix preview.3	
%define blobfuse2_version 2.0.0	
%define blobfuse2_health_monitor bfusemon

Summary:        FUSE adapter - Azure Storage
Name:           blobfuse2
Version:        %{blobfuse2_version}.%{preview_suffix}
Release:        1%{?dist}
License:        MIT
Vendor:         Microsoft Corporation
Distribution:   Mariner
Group:          Applications/Tools
URL:            https://github.com/Azure/azure-storage-fuse/
# Below is the Github URL where blobfuse2 is accessible. This needs to be defined until blobfuse2 GAs since the version
# string in spec files does not allow - 
Source0:        https://github.com/Azure/azure-storage-fuse/archive/%{name}-%{blobfuse2_version}-%{preview_suffix}.tar.gz#/%{name}-%{version}.tar.gz
# Below is a manually created tarball, no download link.
# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
# How to re-build this file:
#   1. wget https://github.com/Azure/azure-storage-fuse/archive/%{name}-%{version}.tar.gz -O %{name}-%{version}.tar.gz
#   2. tar -xf %{name}-%{version}.tar.gz
#   3. cd azure-storage-fuse-%{name}-%{version}
#   4. go mod vendor
#   5. tar  --sort=name \
#           --mtime="2021-04-26 00:00Z" \
#           --owner=0 --group=0 --numeric-owner \
#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
#           -cf %{name}-%{version}-vendor.tar.gz vendor
#
#   NOTES:
#       - You require GNU tar version 1.28+.
#       - The additional options enable generation of a tarball with the same hash every time regardless of the environment.
#         See: https://reproducible-builds.org/docs/archives/
#       - For the value of "--mtime" use the date "2021-04-26 00:00Z" to simplify future updates.
Source1:        %{name}-%{version}-vendor.tar.gz
```